src/components/WebLogView/ResponseDetails/Content.tsx

```diff
@@ -24,6 +24,10 @@ export function Content({ data }: { data: ProxyData }) {
     return reset
   }, [reset])
 
+  useEffect(() => {
+    setSelectedTab('content')
```

Comment:
There's a bug in `main`:
* Open a non-media response
* Switch to `Raw`
* Open a media response - you'll see its raw representation with no way to switch to a normal preview

One can argue that keeping the selected tab state is user friendly - in that case, we can remove that useEffect, it should still work with the changes on line 45

Reply:
I think it's okay for now and we can change if we receive feedback about the usability.